src/Codec/Xlsx/Parser.hs

```diff
@@ -136,69 +149,60 @@ extractCellValue _ "b" "0" = [CellBool False]
 extractCellValue _ _ _ = []
 
 -- | Get xml cursor from the specified file inside the zip archive.
-xmlCursor :: Zip.Archive -> FilePath -> Maybe Cursor
-xmlCursor ar fname = parse <$> Zip.findEntryByPath fname ar
+xmlCursor :: Zip.Archive -> FilePath -> Either ParseError (Maybe Cursor)
+xmlCursor ar fname = maybe (Right Nothing) parse $ Zip.findEntryByPath fname ar
```

Comment:
I haven't used `errors` much but there are some useful utities there which could be handy if `xlsx` will start getting saner error reporting support